pom.xml

```diff
@@ -58,7 +58,7 @@ under the License.
         <netty.version>4.1.86.Final</netty.version>
         <flink.version>1.17.0</flink.version>
         <jackson-bom.version>2.14.3</jackson-bom.version>
-        <glue.schema.registry.version>1.1.14</glue.schema.registry.version>
+        <glue.schema.registry.version>1.1.18</glue.schema.registry.version>
```

Comment:
Upgrading version to resolve the dependency convergence issues with `snakeyaml` package between `flink-core` and `schema-registry-serde`. 
New versions of `schema-registry-serde` no longer include `snakeyaml` as a transitive dependency.

Reply:
Cool. Noted that this module does not have `NOTICE` files so no update needed there.